Makefile

```diff
@@ -349,6 +352,7 @@ PUBLISH_RC_CHART_CMD := bash -c ' \
 	&& mkdir repo \
 	&& cd repo \
 	&& sed -i "s/^version:.*/version: 0.0.1-$(GIT_VERSION)/g" ../open-service-broker-azure/Chart.yaml \
+	&& sed -i "s/^version:.*/version: 0.0.1-$(GIT_VERSION)/g" ../svc-cat-etcd/Chart.yaml \
```

Comment:
Do we want the version of the etcd operator chart tied explicitly to the version of OSBA? I'm thinking not, but am willing to be convinced otherwise.

Is it possible that `Azure/helm-charts` is a better home for this new chart?

The new install docs, however, do clearly belong in this repo. No question there.

Reply:
>Do we want the version of the etcd operator chart tied explicitly to the version of OSBA? I'm thinking not, but am willing to be convinced otherwise.

It's not really the etcd operator chart, it's the chart that creates the cluster _for_ service ctalog...but that doesn't change the fact that the answer is _probably not_. 

> Is it possible that Azure/helm-charts is a better home for this new chart?

Possible. I thought we had kind of turned that repo into something more akin to just examples. I could go either way.

Reply:
> I thought we had kind of turned that repo into something more akin to just examples.

Ya... I've been saying that, haven't I? Maybe I wasn't saying what I meant...

All I wanted to avoid was indiscriminately filling that repo with lots of forks of upstream charts that create additional burden without adding a lot of value. The highest value things sort of ended up being demonstrations of services that we didn't already have demos for-- i.e. examples, as you say.

idk if examples need to be the _only_ use for that repo, though.

In my mind, the best place to keep chart source is close to the app source. (That's why the OSBA chart is here in this repo.) In case, there is no app source (so to speak). The chart just contains resources managed by an operator. That makes me feel a little bit like it's out of place here.

There is, perhaps, an even stronger argument to be made, however, by acknowledging that the etcd this creates is only _indirectly_ a prerequisite of OSBA. The thing most proximately dependent on it is svc cat...

That being the case, I wonder if we should be trying to upstream this to svc cat instead. It may even be that _their_ documentation should be going over "here's how to run a decent etcd in production and here's a chart for it" instead of ours doing that. Perhaps ours should just say "follow the directions for setting up production-grade svc cat."

Thoughts?

Reply:
The only issue I have with trying to upstream this is that it's _very_ Azure specific. AFAIK, the operator only supports AWS and Azure for backup/recovery, so it would be of limited utility I think to the community in general. Happy to open it and see what happens.